openpype/hosts/max/plugins/create/create_review.py

```diff
@@ -53,39 +80,39 @@ def get_instance_attr_defs(self):
                       label="Review width",
                       decimals=0,
                       minimum=0,
-                      default=1920),
+                      default=self.review_width),
             NumberDef("review_height",
                       label="Review height",
                       decimals=0,
                       minimum=0,
-                      default=1080),
-            BoolDef("keepImages",
-                    label="Keep Image Sequences",
-                    default=False),
-            EnumDef("imageFormat",
-                    image_format_enum,
-                    default="png",
-                    label="Image Format Options"),
+                      default=self.review_height),
             NumberDef("percentSize",
                       label="Percent of Output",
                       default=100,
                       minimum=1,
                       decimals=0),
+            BoolDef("keepImages",
+                    label="Keep Image Sequences",
+                    default=self.keep_images),
+            EnumDef("imageFormat",
+                    image_format_enum,
+                    default=self.image_format,
+                    label="Image Format Options"),
             EnumDef("visualStyleMode",
                     visual_style_preset_enum,
-                    default="Realistic",
+                    default=self.visual_style,
                     label="Preference"),
             EnumDef("viewportPreset",
                     preview_preset_enum,
-                    default="Quality",
+                    default=self.viewport_preset,
                     label="Pre-View Preset"),
             EnumDef("antialiasingQuality",
                     anti_aliasing_enum,
                     default="None",
```

Comment:
Shouldn't this take from the defaults as well?
```suggestion
                    default=self.anti_aliasing,
```